src/gen.rs

```diff
@@ -1118,7 +1118,7 @@ pub mod gen {
     #[derive(Clone, Debug, Deserialize, Serialize)]
     #[serde(untagged)]
     pub enum InvokeTxnTraceExecuteInvocation {
-        FunctionInvocation(FunctionInvocation),
+        FunctionInvocation(Box<FunctionInvocation>),
```

Comment:
I know that it is generated code and that it worked fine, I have just made it visible for you so that you can observe what clippy was complaining about. I didn't know that it was already addressed in a different PR, not until I started reviewing it, after this push. I will rebase it.